From c741fe7c090a723c1a0a511fcaa682a9718d3d41 Mon Sep 17 00:00:00 2001
From: "Christopher A. Oliver" <caowasteland@gmail.com>
Date: Sun, 22 Nov 2015 01:22:39 -0500
Subject: [PATCH 0744/1936] Comment out range calculation for modulator
 carrier.  Saved only because someone might want this to bias the carrier in
 future modulator code.

---
 src/Synth/ADnote.cpp | 26 +++++++++++++++-----------
 1 file changed, 15 insertions(+), 11 deletions(-)

diff --git a/src/Synth/ADnote.cpp b/src/Synth/ADnote.cpp
index b1518978..d0df2b5e 100644
--- a/src/Synth/ADnote.cpp
+++ b/src/Synth/ADnote.cpp
@@ -286,17 +286,21 @@ ADnote::ADnote(ADnoteParameters *pars_, SynthParams &spars)
                                              getvoicebasefreq(nvoice),
                                              pars.VoicePar[nvoice].Presonance);
 
-        //Find range of generated wave
-        float min = NoteVoicePar[nvoice].OscilSmp[0];
-        float max = min;
-        float *smpls = &(NoteVoicePar[nvoice].OscilSmp[1]);
-        for (int i = synth.oscilsize-1; i--; smpls++)
-            if (*smpls > max)
-                max = *smpls;
-            else if (*smpls < min)
-                min = *smpls;
-        NoteVoicePar[nvoice].OscilSmpMin = min;
-        NoteVoicePar[nvoice].OscilSmpMax = max;
+        // This code was planned for biasing the carrier in MOD_RING
+        // but that's on hold for the moment.  Disabled 'cos small
+        // machines run this stuff too.
+        //
+        // //Find range of generated wave
+        // float min = NoteVoicePar[nvoice].OscilSmp[0];
+        // float max = min;
+        // float *smpls = &(NoteVoicePar[nvoice].OscilSmp[1]);
+        // for (int i = synth.oscilsize-1; i--; smpls++)
+        //     if (*smpls > max)
+        //         max = *smpls;
+        //     else if (*smpls < min)
+        //         min = *smpls;
+        // NoteVoicePar[nvoice].OscilSmpMin = min;
+        // NoteVoicePar[nvoice].OscilSmpMax = max;
 
         //I store the first elments to the last position for speedups
         for(int i = 0; i < OSCIL_SMP_EXTRA_SAMPLES; ++i)
-- 
2.47.0

